proto/frequenz/api/weather/weather.proto

```diff
@@ -97,6 +98,11 @@ message GetHistoricalWeatherForecastRequest {
   // same request must include the 'page_token' from the previous response.
   frequenz.api.common.v1.pagination.PaginationParams
     pagination_params = 5;
+
+  // The validity times for which forecasts are being requested. These values
+  // should be relative to the generation time of the forecasts. When none are
+  // provided, forecasts for all available validity times will be returned.
+  repeated google.protobuf.Duration validity_times = 6;
```

Comment:
I think it would be great to provide an example of what the validity time means. Right now I do not understand the usage of it.

Reply:
> "These values should be relative"

Should be? Are they, or are they not? The meaning is confusing.

Reply:
Forecasts generated at any time will have multiple forecasts for each feature, one valid for each hour for the next 24 hours (or upto 2 weeks).  I will add a few lines.

>> "These values should be relative"
>
> Should be? Are they, or are they not? The meaning is confusing.

These are the request objects coming from the client.  When the client is creating the request, they should use validity times relative to the forecast time.  We only accept objects of type `Duration`, so that makes the same point clear as well I think.